tests/v2/validation/rbac/rbacuserroles/project_role_test.go

```diff
@@ -0,0 +1,129 @@
+	require.NoError(pr.T(), err)
+
+	createProjectAsAdmin, err := rbac.CreateProject(pr.client, pr.cluster.ID)
+	require.NoError(pr.T(), err)
+
+	log.Info("Adding a standard user as project Owner in the admin project")
+	errUserRole := users.AddProjectMember(pr.client, createProjectAsAdmin, newUser, rbac.RoleProjectOwner, nil)
+	require.NoError(pr.T(), errUserRole)
+	standardUserClient, err = standardUserClient.ReLogin()
+	require.NoError(pr.T(), err)
+	return standardUserClient, createProjectAsAdmin
+}
+
+func (pr *ProjectRolesTestSuite) TestProjectOwnerAddsRemovesOtherProjectOwners() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	standardUserClient, adminProject := pr.testSetupUserAndProject()
+	additionalUser, additionalUserClient, err := rbac.SetupUser(pr.client, rbac.StandardUser)
+	require.NoError(pr.T(), err)
+
+	log.Info("Validating if Project Owner can add another Project Owner")
+	errUserRole := users.AddProjectMember(standardUserClient, adminProject, additionalUser, rbac.RoleProjectOwner, nil)
+	require.NoError(pr.T(), errUserRole)
+	additionalUserClient, err = additionalUserClient.ReLogin()
+	require.NoError(pr.T(), err)
+
+	userGetProject, err := projects.ListProject(additionalUserClient, pr.cluster.ID)
+	require.NoError(pr.T(), err)
+	assert.Equal(pr.T(), 1, len(userGetProject.Data))
+	assert.Equal(pr.T(), adminProject.Name, userGetProject.Data[0].Name)
+
+	errRemoveMember := users.RemoveProjectMember(standardUserClient, additionalUser)
+	require.NoError(pr.T(), errRemoveMember)
+
+	userProjectEmptyAfterRemoval, err := projects.ListProject(additionalUserClient, pr.cluster.ID)
+	require.NoError(pr.T(), err)
+	assert.Equal(pr.T(), 0, len(userProjectEmptyAfterRemoval.Data))
+}
+
+func (pr *ProjectRolesTestSuite) TestManageProjectUserRoleCannotAddProjectOwner() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	standardUserClient, adminProject := pr.testSetupUserAndProject()
+	additionalUser, additionalUserClient, err := rbac.SetupUser(pr.client, rbac.StandardUser)
+	require.NoError(pr.T(), err)
+
+	log.Info("Validating Manage Project Member cannot add Project Owner")
+	errUserRole := users.AddProjectMember(standardUserClient, adminProject, additionalUser, rbac.RoleCustomManageProjectMember, nil)
+	require.NoError(pr.T(), errUserRole)
+	additionalUserClient, err = additionalUserClient.ReLogin()
+	require.NoError(pr.T(), err)
+
+	addNewUserAsPO, addNewUserAsPOClient, err := rbac.SetupUser(pr.client, rbac.StandardUser)
+	require.NoError(pr.T(), err)
+
+	errUserRole2 := users.AddProjectMember(additionalUserClient, adminProject, addNewUserAsPO, rbac.RoleProjectOwner, nil)
+	require.Error(pr.T(), errUserRole2)
+	errStatus := strings.Split(errUserRole2.Error(), ".")[1]
+	rgx := regexp.MustCompile(`\[(.*?)\]`)
+	errorMsg := rgx.FindStringSubmatch(errStatus)
+	assert.Equal(pr.T(), "422 Unprocessable Entity", errorMsg[1])
```

Comment:
We can use assert.Contains? In this case we dont have to use rgx.

Is this error(422 Unprocessable Entity) correct? I am not sure but just checking.

Reply:
sure we can user assert.Contains. 

Yes thats the error 422 we are seeing currently.